checker/worker/worker.go

```diff
@@ -43,7 +43,18 @@ func (worker *Checker) Start() error {
 
 	worker.lastData = time.Now().UTC().Unix()
 
-	metricEventsChannel, err := worker.Database.SubscribeMetricEvents(&worker.tomb)
+	if worker.Config.MetricEventPopBatchSize == 0 {
+		worker.Config.MetricEventPopBatchSize = 100
+	} else if worker.Config.MetricEventPopBatchSize < 0 {
+		return errors.New("MetricEventPopBatchSize param less than zero")
+	}
+
+	subscribeMetricEventsParams := moira.SubscribeMetricEventsParams{
+		BatchSize: int64(worker.Config.MetricEventPopBatchSize),
```

Comment:
maybe [here](https://github.com/moira-alert/moira/pull/802/files#diff-a1c3b1ad9fc3f8e561060748f53a8cee5af1448c3f02bcf07537263d26d256e8R19) int64?

Reply:
to not cast in each loop)

Reply:
There is no loop

Reply:
aaaaa, sooorry(( my fault